feat: FIT-670: Improve guidance and organization of Annotation settings page

ricardoantoniocm opened the pull request:
This pull request adds two new SVG icon exports to the `web/libs/ui/src/assets/icons/index.ts` file, making them available for use throughout the application.

**New icon exports:**

* Added `IconAutomatic` export for `automatic.svg` to enable usage of the automatic icon.
* Added `IconUserEdit` export for `user-edit.svg` to enable usage of the user edit icon.

<!--

This description MUST be filled out for a PR to receive a review. Its primary purposes are:

 - to enable your reviewer to review your code easily, and
 - to convince your reviewer that your code works as intended.

Some pointers to think about when filling out your PR description:
 - Reason for change: Description of problem and solution
 - Screenshots: All visible changes should include screenshots.
 - Rollout strategy: How will this code be rolled out? Feature flags / env var / other
 - Testing: Description of how this is being verified
 - Risks: Are there any known risks associated with this change, eg to security or performance?
 - Reviewer notes: Any info to help reviewers approve the PR
 - General notes: Any info to help onlookers understand the code, or callouts to significant portions.

You may use AI tools such as Copilot Actions to assist with writing your PR description (see https://docs.github.com/en/copilot/using-github-copilot/using-github-copilot-for-pull-requests/creating-a-pull-request-summary-with-github-copilot); however, an AI summary isn't enough by itself. You'll need to provide your reviewer with strong evidence that your code works as intended, which requires actually running the code and showing that it works.

-->

ricardoantoniocm committed:
feat: FIT-670: Improve guidance and organization of Annotation settings page

ricardoantoniocm committed:
Adds Automatic and User Edit icons.

ricardoantoniocm committed:
Updates automatic icon.

ricardoantoniocm committed:
Fixes play pictogram inside automatic icon.

ricardoantoniocm commented:
/git merge
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/17765349570)
> Successfully merged:  create mode 100644 web/libs/ui/src/assets/icons/cloud-provider-databricks.svg

ricardoantoniocm committed:
Merge branch 'develop' into 'fb-fit-670'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/17765349570

borisheartex commented:
/fm sync
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/17789981555)

borisheartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/17789981555

borisheartex committed:
Merge branch 'develop' into 'fb-fit-670'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/17789981555

ricardoantoniocm commented:
/git merge
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/17835832369)
> Successfully merged:  delete mode 100644 web/libs/datamanager/src/components/Common/Form/Elements/RadioGroup/RadioGroup.scss

ricardoantoniocm committed:
Merge branch 'develop' into 'fb-fit-670'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/17835832369
